components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/view/ImporterUIElementLight.java

```diff
@@ -164,10 +167,24 @@ private void buildGUI() {
         c.gridx = 0;
         c.gridy = 3;
         c.fill = GridBagConstraints.NONE;
+        cancelledLabel.setVisible(false);
+        info.add(cancelledLabel, c);
+        
+        c.gridx = 1;
+        c.gridy = 3;
+        c.fill = GridBagConstraints.HORIZONTAL;
+        c.gridwidth = 2;
+        c.anchor = GridBagConstraints.WEST;
```

Comment:
Right-justifies the labels (addresses Marks [comment](https://github.com/openmicroscopy/openmicroscopy/pull/5783/commits/144893ef5090a2c443c10fa95d9f859a4081bf0a#r206153748) )